src/bootstrap/server-bootstrapper.js

```diff
@@ -46,6 +47,12 @@ export async function createContainer(configValues = {}) {
 
   logger.info('Loading resources');
   const additionalResources = await Promise.all(serverConfig.resources.map(async modulePath => {
+    if ((modulePath.toLowerCase()).endsWith('.json')) {
+      const resourceFileUrl = new URL(modulePath, import.meta.url);
+      const resourceJSON = JSON.parse(readFileSync(resourceFileUrl));
+      return resourceJSON;
+    }
+
```

Comment:
As we do not have any other resources, we do not need the `if` and also not the code below this comment.

I wonder why we can still import the "main" resources with the import assertion but not the additional resources. What was the error there?

Reply:
So maybe vitest cannot parse dynamic imports with assertions (because static imports seem to work fine). Thanks.